apps/serialization/gateway.proto

```diff
@@ -3,7 +3,6 @@ syntax = "proto3";
   message WebSocketRequest {
     oneof request_type {
       GetUser get_user = 1;
-      GetUserByUsername get_user_by_username = 2;
```

Comment:
Why isn't the index of PB updated?

Reply:
We should never change the indexing number of the .proto to keep consistency with previous/future versions of the file, that's the same reason why you always add a new field with the latest index instead of moving them.